robot.py

```diff
@@ -97,6 +97,14 @@ def createObjects(self):
         self.chassis_3_encoder = ctre.CANCoder(3)
         self.chassis_4_encoder = ctre.CANCoder(4)
 
+    def robotInit(self):
+        wpilib.CameraServer.launch()
+        return super().robotInit()
```

Comment:
Wouldn't you want to do this in `createObjects`?
```suggestion
        wpilib.CameraServer.launch()
```